libsnark/zk_proof_systems/ppzksnark/r1cs_gg_ppzksnark/r1cs_gg_ppzksnark.tcc

```diff
@@ -313,7 +306,7 @@ r1cs_gg_ppzksnark_keypair<ppT> r1cs_gg_ppzksnark_generator(const r1cs_gg_ppzksna
     libff::leave_block("Generating G1 MSM window table");
 
     libff::enter_block("Generating G2 MSM window table");
-    const libff::G2<ppT> G2_gen = libff::G2<ppT>::random_element();
+    const libff::G2<ppT> G2_gen = libff::G2<ppT>::one();
```

Comment:
`G2_gen` -> `g2_generator` to keep the naming consistent with the generator of g1.
I think we can pass it as an argument to the function and remove this line